source/_patterns/20-molecules/teaser/teaser.scss

```diff
@@ -456,7 +456,7 @@
       flex-wrap: wrap;
     }
 
-    // if XL-Teaser is beeing clicked … do the following with it's siblings and children
+    // if XL-Teaser is being clicked … do the following with it's siblings and children
```

Comment:
its = possessive pronoun (a duck and its children)
it's = short for `it is` (it's a plane)

http://www.its-not-its.info/

Reply:
HAH, ja eigentlich logisch :D